package.json

```
@@ -70,15 +74,16 @@
     "compile": "run-p compile:*",
     "compile:prisma": "yarn --cwd server build:prisma",
     "compile:pathpida": "pathpida",
-    "build": "run-s compile build:frontend build:export build:server",
-    "build:frontend": "run-p build:{storybook,next}",
+    "build": "run-s compile build:static build:export build:server",
+    "build:static": "run-p build:{license,storybook,next}",
+    "build:license": "yarn licenses generate-disclaimer > public/LICENSE",
```

```suggestion
    "build:license": "NODE_ENV=development yarn licenses generate-disclaimer > public/LICENSE",
```

このようにすれば依存関係にあるパッケージのライセンス表記も含まれるでしょうか

productionではdevDependenciesのパッケージのライセンス表記を含まないようですね https://github.com/yarnpkg/yarn/pull/2941

---

問題の考察と変更の提案ありがとうございます! 良さそうに思います!